crates/typst/src/layout/grid/layout.rs

```diff
@@ -508,14 +388,14 @@ impl<'a> GridLayouter<'a> {
     pub fn layout(mut self, engine: &mut Engine) -> SourceResult<GridLayout> {
```

Comment:
I think we can delete the GridLayout type and return a Fragment directly. It was only necessary to return information about rows and columns which were needed for stroke drawing.

Reply:
Fair enough. I actually considered using `GridLayout` to avoid mutability problems when rendering fills and strokes, but I managed to avoid those using `self.grid`, so this should be safe. We can always reintroduce the type if this is eventually needed again.